source/Ship.cpp

```diff
@@ -1439,18 +1439,18 @@ int Ship::Scan()
 	if(government->IsPlayer() || (target->GetGovernment()->IsPlayer() && activeScanning))
 		Audio::Play(Audio::Get("scan"), Position());
 	
-	if(startedScanning && government->IsPlayer())
+	if(startedScanning && government->IsPlayer() && !target->GetGovernment()->IsPlayer())
```

Comment:
Oh, you were referring to initiating a scan. Sorry, I glossed over that aspect. The outcome of the scan should not include the Escort government, though.